changes/issue-8344-add-arch-mint-linux-distros

```diff
@@ -0,0 +1 @@
+* Added arch and linuxmint to allow list so that their data is displayed and host count includes them
```

Comment:
We don't really refer to this as an allow list. Rather, this is the list of platforms we identify as being "linux".